Calculator/Calculator.cpp

```diff
@@ -82,97 +84,147 @@ Token Token_stream::get()
 
 //------------------------------------------------------------------------------
 
-Token_stream ts;       // provides get() and putback()
+Token_stream ts;       
 
 //------------------------------------------------------------------------------
 
-double expression();  // read and evaluate a Expression
+double expression();  
 
```

Comment:
why are these here? lol don't worry about commenting these now just consider if you should or not